www/templates/diary/trip_list_item.html

```diff
@@ -62,15 +62,15 @@
 
                     <!-- Row, containing Detail Screen button and Trip Origin/Destination -->
                     <div class="row" style="padding: 4px">
-                        <div class="col-100">
+                        <div class="col-100" style="width: 100%;">
                             <!-- Origin for Trip -->
-                            <div class="diary-street" ng-click="showDetail($event, trip)">
+                            <div class="diary-street two-lines" ng-click="showDetail($event, trip)">
```

Comment:
I like the fact that you put this into a class instead of bulking up the style attribute!